CSharp/README.md

```diff
@@ -206,6 +207,71 @@ var result=2;
 var result = 2;
 ```
 
+## Ordering
+
+### Within a class, struct, or interface, elements must be ordered
+
+1. Constants
+1. Fields
+1. Constructors
+1. Delegates
+1. Events
+1. Properties
+1. Methods
+1. Finalizers (Destructors)

```

Comment:
We could. I will try to add in the description but not sure if you will like it 😄